### ISSUE TITLE
Review a11y for Text field component

### ISSUE THREAD

**Issue by awentzel** [Microsoft/fast-dna#172]
### Expected behavior 
Component audit to ensure grade C accessibility. Make updates if they are required.  No component should be released without accessibility hooked up.  Should be validated based on the spec https://www.w3.org/TR/wai-aria-practices-1.1/.

Each base component must pass accessibility by a third party (@Wilto  or CELA).

Upon approval and success, this task can be closed.

**Comment by Falkicon** [comment]
@awentzel , Assigning to you to figure out what to do with this task. For example, is this task to do the audit, arrange the audit. It was assigned to Eric who I would not assume would be responsible for either. I would expect that upon completion of this task, there may be some additional tasks created to address accessibility.

**Comment by awentzel** [comment]
Updated comments.

**Comment by awentzel** [comment]
Scheduling a meeting to discuss the UX on https://msft-docs.azurewebsites.net/components/text-field 

It feels broken when using the configurator and thus is blocking the ability to schedule a11y test of this component or it's different input types.

Also, should discuss the name.  Feels more like an "Input" as you can choose one of many different "Input types".  So not clear why it would be "Text field" if that is only a single input type?

**Comment by awentzel** [comment]
After reviewing with fellow collaborators it was determined there are a few things remaining before this is ready for an a11y audit. 
1. "Text field" should only include input types for strings and numbers.  Meaning that date and range really should be a new component.
2.  The configurator should be fixed so that it doesn't feel broken to users. I would expect that placeholder text gets replaced with specific text for each of the different input types.  When selecting an input type the placeholder text should replace and the code preview should update.
3. Additional, test pass should be made after these issues are resolved.

**Comment by awentzel** [comment]
Issue #721 #722  created to bring Text-field to a place where it can be reviewed.

**Comment by chrisdholt** [comment]
Closing this as #721 and #722 were created in place of this. As those have been closed I am closing this.
